data/transactions/logic/evalStateful_test.go

```diff
@@ -787,6 +804,7 @@ func testApp(t *testing.T, program string, ep EvalParams, problems ...string) ba
 		require.NoError(t, err)
 		require.Empty(t, delta.GlobalDelta)
 		require.Empty(t, delta.LocalDeltas)
+		require.Empty(t, delta.LogDelta)
```

Comment:
"Logs" is probably a better name than "LogDelta" since there is no "change" here, implied by the "Delta" naming.